modules/shadertools/src/modules/geometry/geometry.js

```diff
@@ -1,20 +1,18 @@
 // Cheap lighting - single directional light, single dot product, one uniform
 
 /* eslint-disable camelcase */
-const DEFAULT_MODULE_OPTIONS = {};
-
-function getUniforms(opts = DEFAULT_MODULE_OPTIONS) {
-  const uniforms = {};
-  return uniforms;
-}
 
 // TODO - reuse normal from geometry module
 const vs = `\
-varying vec4 geometry_vPosition;
-varying vec3 geometry_vNormal;
+#version 300 es
+
+out vec4 geometry_vPosition;
+out vec4 geometry_vNormal;
 
 void geometry_setNormal(vec3 normal) {
-  geometry_vNormal = normal;
+  geometry_vNormal.xyz = normal;
+  geometry.w = 0.;
```

Comment:
This is meant to be `geometry_vNormal`?

I don't think we can assume this will start at 1.

Reply:
You are probably right. I seem to find guarantees around uniform initial values (0, 0, 0, 0) and attributes (0, 0, 0, 1) but so far nothing around varyings.